test/adding_tests.md

```diff
@@ -175,15 +189,17 @@ err := test.WaitForConfigurationState(clients.ServingClient, configName, func(c
 }, "ConfigurationUpdatedWithRevision")
 ```
 
-_[Metrics will be emitted](#emit-metrics) for these `Wait` method tracking how
+_v1alpha1testing is alias for package `knative.dev/serving/pkg/testing/v1alpha1`_
+
+_Metrics will be emitted for these `Wait` method tracking how
```

Comment:
We removed the metrics, your can delete this paragraph.

Reply:
done